README.rst

```diff
@@ -24,11 +24,12 @@ You can install with `pip <https://pip.pypa.io/en/latest/>`__.
 
     $ pip install vim-vint
 
-If you want to install development version
+If you want to install the development version
 
 ::
 
-    $ pip install git+https://github.com/Kuniwak/vint
+    $ git clone https://github.com/Kuniwak/vint
+    $ setup.py install
```

Comment:
Hmm, the first variant is actually better/easier, no?
(this one would also need a `cd vint`)

Reply:
You are right.

How's this?

```
pip install git+https://github.com/Kuniwak/vint
```
or
```
$ git clone https://github.com/Kuniwak/vint
$ cd vint
$ setup.py install
```

So feel free to close it in case you do not want to adopt it.